lib/membership_pair_entry.dart

```diff
@@ -0,0 +1,6 @@
+class MembershipPairEntry {
+  final String membershipPairId;
+  final bool older;
```

Comment:
What's older about?

Reply:
this is to know which member came first, so we know which peer should initiate the offer and which should wait for it

might not be ideal though